src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double width;
+    private double height;
+
+    public Rectangle(Color color, double width, double height) {
+        super(color, width * height);
+        this.width = width;
+        this.height = height;
+    }
```

Comment:
Creating a figure, don't pass expressions in the constructor. Instead, create separate variables for width * height and pass them to the super constructor for better code readability.